src/pages/authorization/sign-up/sign-up.validation.js

```diff
@@ -0,0 +1,24 @@
+export const requirePassword = value =>
+  value ? undefined : "Please type your password";
```

Comment:
I would rather use "Please enter a password", "Please enter an email address", "Please repeat your password"(for the second password field).